src/Compilers/CSharp/Portable/Binder/LocalBinderFactory.cs

```diff
@@ -199,6 +199,45 @@ public override void VisitOperatorDeclaration(OperatorDeclarationSyntax node)
             Visit(node.ExpressionBody);
         }
 
+        public override void VisitInvocationExpression(InvocationExpressionSyntax node)
+        {
+            if (node.MayBeNameofOperator())
+            {
+                WithTypeParametersBinder nextWhenNameofOperator = ((_enclosing.Flags & BinderFlags.InContextualAttributeBinder) != 0) && MessageID.IDS_FeatureExtendedNameofScope.IsFeatureAvailable(node)
```

Comment:
>nextWhenNameofOperator [](http://example.com/codeflow?start=41&length=22)

The terminology  feels very confusing. Consider renaming, see relevant comments for NameofBinder.